src/groupByKey.ts

```diff
@@ -2,6 +2,23 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(
+  items: Array<object>,
+  key: string,
```

Comment:
```suggestion
  key: keyof T,
```

it would be better to convert key to string when you iterate through the loop